README.md

```diff
@@ -1415,6 +1415,28 @@ which will halt execution.
   `requirement`, e.g., `">=0.1.0"`, returning `"true"` if so and `"false"`
   otherwise.
 
+##### Directory Locaters
+
+- `cache_directory()` - Locate the user-specific cache directory. Returns an
```

Comment:
Let's call these `cache_dir` `config_dir`, etc.

Reply:
I think it is good to have consistency with `invocation_directory` `justfile_directory` and `parent_directory`, which is why I did it this way. If you prefer `_dir` here, would you be okay adding aliases to the existing functions as e.g. `justfile_dir`?

 I get confused with APIs that alternate naming :)

Reply:
It is a bit confusing, but I think most people won't wind up looking at the `dirs` docs, so being out of sync won't be a problem.

Reply:
I think I'm misunderstanding - are you agreeing with my statement above?

My suggestion was either to leave the PR as-is so Just's functions are `justfile_directory`, `config_directory`, `cache_directory` etc (consistent within Just but inconsistent with `dirs` is fine), or to use `config_dir` but also add aliases so `justfile_dir`, `invocation_dir`, `parent_dir` all work. As opposed to having `justfile_directory` but `cache_dir`, and maybe needing to look up which is which.

Do either of these sound better to you?

Reply:
Ah yeah, I see what you're saying. In that case, let's us `_directory`, and later I might open a PR which adds `_dir` aliases for everything.

Reply:
That works for me 👍